opentreemap/treemap/js/src/lib/geocoderUi.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
I just tried this in Firefox and I did not see the error, but I did see that the search by location box no longer has the dark background.